src/feed-queue.js

```diff
@@ -1,5 +1,7 @@
+const config = require('./config');
 const Bull = require('bull');
 
-const queue = new Bull('feed-queue');
+const redisUrl = process.env.REDIS_URL;
```

Comment:
this line can be removed and replace line 5 with 
const queue = new Bull('feed-queue', process.env.REDIS_URL);

Reply:
Combined the lines to appear as requested.